src/containers/ConversionForm/index.tsx

```diff
@@ -70,21 +71,28 @@ const ConversionForm = ({
     receiveFileToConvert,
     initializeConversion,
     conversionStatus,
+    convertFile,
+    setConversionStatus,
 }: ConversionProps): JSX.Element => {
     const [fileToConvert, setFileToConvert] = useState<UploadFile | null>();
     const [engineSelected, setEngineSelected] = useState<boolean>(false);
```

Comment:
Also not technically part of this PR (so feel free to ignore me!), but this looks like derived state to me. `engineSelected` tracks whether another component of state is set, and may become out of sync if not properly updated (e.g. `engineSelected: false` while `store.trajectory.processingData.engineType: "cytosim"`). This particular case is probably too simple to cause issues, but it still might be slightly better practice to select `engineType` out of the store and null-check it.

Reply:
Love it, seems like this works and can remove the state variable:
`const engineSelected = !!conversionProcessingData.engineType;`